tfjs-backend-webgpu/src/kernels/crop_and_resize_webgpu.ts

```diff
@@ -79,16 +80,11 @@ export class CropAndResizeProgram implements WebGPUProgram {
     // tslint:disable-next-line:max-line-length
     // https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/crop_and_resize_op_gpu.cu.cc
     const userCode = `
-      fn writeResult(coords : vec4<i32>, value : f32) {
-        if (coordsInBounds4D(coords, uniforms.outShape)) {
-          setOutput(coords[0], coords[1], coords[2], coords[3], value);
-        }
-      }
-      ${getMainHeaderString()} {
-        ${getGlobalIndexString()}
+      ${getMainHeaderAndGlobalIndexString()}
+      if (index < uniforms.size) {
```

Comment:
Does coordsInBounds4D equal index < uniforms.size?

Reply:
Yes, they are equal since `coords` is never changed.